pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -91,12 +87,30 @@ func validateClusterServiceBrokerSpec(spec *sc.ClusterServiceBrokerSpec, fldPath
 		}
 	}
 
+	commonErrs := validateCommonServiceBrokerSpec(&spec.CommonServiceBrokerSpec, fldPath)
+
+	if len(commonErrs) != 0 {
+		allErrs = append(commonErrs)
+	}
+
+	return allErrs
+}
+
+func validateCommonServiceBrokerSpec(spec *sc.CommonServiceBrokerSpec, fldPath *field.Path) field.ErrorList {
+	commonErrs := field.ErrorList{}
```

Comment:
why rename?

Reply:
it matches the naming of the spec object being validated.